actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -358,7 +358,7 @@ def authorization
       get_header('REDIRECT_X_HTTP_AUTHORIZATION')
     end
 
-    # True if the request came from localhost, 127.0.0.1.
+    # True if the request came from localhost, 127.0.0.1, ::1.
```

Comment:
Thanks for the PR! :smile:

Can you update this to say "True if the request came from localhost, 127.0.0.1, or ::1." to be more grammatically correct?